Client.py

```diff
@@ -208,6 +225,8 @@ def runMenu(self):
         elif self.menu.showMenu() == MenuState.LOBBY:
             menuState = MenuState.LOBBY
         if self.allPlayersJoined():
+            self.startCountdown()
+            self.shouldLoadSong = True
```

Comment:
You already set `self.shouldLoadSong` to True in `startCountdown()`

Reply:
done